lib/views/scannerview.dart

```diff
@@ -30,122 +30,135 @@ class ScanScreenState extends State<ScanScreen> {
     super.dispose();
   }
 
-  Future<void> _startBarcodeScan() async {
+
+
+
+  Future<void> _startBarcodeScan(String barcodeScanRes) async {
     try {
-      String barcodeScanRes = await FlutterBarcodeScanner.scanBarcode(
-        '#ff6666',
-        'Cancel',
-        true,
-        ScanMode.BARCODE,
-      );
       if (barcodeScanRes != '-1') {
```

Comment:
scannerview : Remove Barcodescanner for mobile 

As the application is going to run on windows we can remove this implementation